src/shivangi/exam/Shivangi_Exam2.java

```diff
@@ -0,0 +1,48 @@
+/*
+1. How to find all pairs in an array of integers whose sum is equal to the given number
+i/p  : {5,1,2,7,6,3,4,9} , sum : 6
+o/p : {5,1}, {2,4}
+
+2. WAP to print a single occurrence element from the array in Java?
+i/p  : {5,8,1,2,4,3,2,1}
+o/p : {5,8,4,3}
+*/
+package shivangi.exam;
+
+public class Shivangi_Exam2 {
+	void findSumOfPairs() {
+		int[] arr = {5,1,2,7,6,3,4,9};
+		int sum = 6;
+		System.out.println("Pairs whose sum is equal to "+sum+" : ");
+		for(int index=0; index<arr.length; index++) {
+			for(int innerIndex=0; innerIndex<arr.length; innerIndex++) {
```

Comment:
this inner index should starts from index+1; think on this @ShivangiTechnocredits